Assets/Shopify/UIToolkit/Shops/Interfaces/IShop.cs

```diff
@@ -58,5 +58,12 @@ public interface IShop {
         /// </summary>
         /// <param name="newQuantity">The new number of items in the cart</param>
         void OnCartQuantityChanged(int newQuantity);
+
+        /// <summary>
+        /// Called when anything changes with the items that are in the cart. For example,
+        /// adding/removing/updating an item in your cart will trigger this.
+        /// </summary>
+        /// <param name="cartItems">List of cart items in the cart</param>
+        void OnCartItemsChanged(List<CartItem> cartItems);
```

Comment:
Wanna move this + OnCartQuantity changed to the `IMultiProductShop` interface? WDYT? does single product shop need a cart?

Reply:
If we're saying a single product shop is like a 'buy this thing now' kind of store then we can remove. If we want to be able to add multiple of a single item then we'll probably need a cart. I say we should just remove it for now and go with the first option.